packages/node-bridge/web-handler.ts

```diff
@@ -0,0 +1,27 @@
+import type { BuildDependencies } from '@edge-runtime/node-utils';
+import { buildToNodeHandler } from '@edge-runtime/node-utils';
+
+export function transformToNodeHandler(webHandler: any) {
```

Comment:
**question (non-blocking):** `transformToNodeHandler` is often called with a second paramter (`runtime`), but that's not used. Should it be used in some way or removed from call sites?